dbunit/src/test/java/com/baeldung/dbunit/OldSchoolDbUnitTest.java

```diff
@@ -0,0 +1,111 @@
+package com.baeldung.dbunit;
+
+import org.dbunit.Assertion;
+import org.dbunit.IDatabaseTester;
+import org.dbunit.JdbcDatabaseTester;
+import org.dbunit.dataset.IDataSet;
+import org.dbunit.dataset.ITable;
+import org.dbunit.dataset.xml.FlatXmlDataSet;
+import org.dbunit.operation.DatabaseOperation;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.io.InputStream;
+import java.sql.Connection;
+import java.sql.ResultSet;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class OldSchoolDbUnitTest {
```

Comment:
What makes this `OldSchool` - can you add javadoc to explain how it differs from `SampleDBUnit` and can you think of a name that characterises the difference between the options, rather than how old-fashioned a pattern it is.